ibis/backends/polars/compiler.py

```diff
@@ -1521,3 +1521,10 @@ def visit_ArrayContains(op, **kw):
     arg = translate(op.arg, **kw)
     value = translate(op.other, **kw)
     return arg.list.contains(value)
+
+
+@translate.register(ops.ArrayRemove)
+def visit_ArrayRemove(op, **kw):
+    arg = translate(op.arg, **kw)
+    value = _literal_value(op.other)
```

Comment:
Based on the documentation it could be a non-literal as well https://docs.pola.rs/api/python/stable/reference/series/api/polars.Series.list.set_difference.html 

I can see that you tried that as well but had some problems, could you create a follow-up ticket for it?

Reply:
Thanks for the review! I have created https://github.com/ibis-project/ibis/issues/10326 so that this can be reviewed/discussed in more detail.